.gitmodules

```diff
@@ -45,3 +45,6 @@
 [submodule "deps/si_units"]
 	path = deps/si_units
 	url = https://github.com/HeisenbugLtd/si_units.git
+[submodule "deps/clic"]
+	path = deps/clic
+	url = git@github.com:alire-project/clic
```

Comment:
I suspect this git@ URL won't work for people outside of the alire-project org.

Reply:
This should require and use the SSH key of the user. Better change it to `https://github.com/alire-project/clic`? :slightly_smiling_face:

Reply:
Will fix. I had to do it that way when the repo was private.